### ISSUE TITLE
Add Texture Filter Inherit to Elements of GridMap and Sprite3D

### ISSUE THREAD

**Issue by Daniel-da-Silva** [godotengine/godot-proposals#6756]
### Describe the project you are working on

3D game with Pixelart

### Describe the problem or limitation you are having in your project

Everytime I make a new Sprite3D I must manually set the texture filter to nearest.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

I would like to have an Inherit flag, so you can make it project wide, this would speed up development.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

![inherit](https://user-images.githubusercontent.com/73739524/234217919-6472b021-9b45-44bf-a07b-8b73baa3e04b.png)

### If this enhancement will not be used often, can it be worked around with a few lines of script?

Will be used often for 3D Pixelart games

### Is there a reason why this should be core and not an add-on in the asset library?

Basically the example text: This is about improving the 3D editor usability out of the box.

**Comment by Calinou** [comment]
- Related to https://github.com/godotengine/godot-proposals/issues/5228.

The way filtering is set in 3D doesn't make it possible to achieve this like in 2D. All we can do is add a "Default" option which matches a project-wide default.

**Comment by Daniel-da-Silva** [comment]
That would be great Hugo, a default option like you described is exactly what I am looking for!

**Comment by Calinou** [comment]
- Duplicate of https://github.com/godotengine/godot-proposals/issues/5228 (I've modified it to mention SpriteBase3D).
